quickstart/armv7a9-zynq7000-zturn.md

```diff
@@ -0,0 +1,245 @@
+# Running system on `armv7a9-zynq7000-zturn` (Xilinx Zynq-7000, Zturn development board)
+
+These instructions describe how to run a Phoenix-RTOS system image for `armv7a9-zynq7000-zturn` target architecture.
+Note that, the build artifacts, including the system image, should be first provided in the `_boot` directory.
+If you haven't run the `build.sh` script yet, run it for `armv7a9-zynq7000-zturn` target.
+
+See [how to build the Phoenix-RTOS system image](../building/README.md).
+
+## Preparing the board
+
+Preparing the board depends on how the plo is loaded into RAM, this quickstart describes 2 approaches - loading from SD
+card and NAND flash, depending on your needs use one of them. For example if you have empty flash memory or want to
+load new plo use SD card, otherwise you can simply load plo from NAND flash.
+
+### Load plo from SD card
+
+- Firstly, prepare SD card, I would prefer to format it to FAT32
```

Comment:
Please remove this point and add a link to common problems as it's done in zedboard quickstart + eventually add a note about FAT32 over there.

```suggestion
```

Reply:
Where is this link? I would prefer to place:

```If you encountered some problems during this step please see
 [common problems](armv7a9-zynq7000.md#common-problems-on-zynq7000-boards).```

just before the following chapter as it's done for zedboard:

`Erasing the area intended for file system`